### PR TITLE
fixes rtx double stop panic

### PR DESCRIPTION
**Pull request by andrewpmartinez**
- stopping status was being reset causing subsequent stops to panic
- remove stopping atomic bool
- change router disconnect logging from debug to info

### CODE REVIEW

File: controller/sync_strats/rtx.go [hunk 1]
```diff
@@ -66,22 +64,26 @@ func (rtx *RouterSender) GetState() env.RouterStateValues {
 
 func (rtx *RouterSender) Start() {
 	if rtx.running.CompareAndSwap(false, true) {
```
Comment (plorenz) [inline controller/sync_strats/rtx.go:66]: I'd leave closeNotify init above. I don't see a reason to create it just-in-time or nil it out afterwards. Maybe I'm missing something, though?

Comment (andrewpmartinez) [reply controller/sync_strats/rtx.go:66]: It was to allow the `RouterSender` to be re-stoppable and re-startable. I can make it not that as that functionality isn't used in the final version.

Comment (plorenz) [reply controller/sync_strats/rtx.go:66]: I'd wait until we're using it to add that, just in case we never do :)

Comment (andrewpmartinez) [reply controller/sync_strats/rtx.go]: Removed.

File: controller/sync_strats/rtx.go [hunk 2]
```diff
@@ -38,22 +38,24 @@ type RouterSender struct {
...
-	return &RouterSender{
+	rtx := &RouterSender{
 		Id:          eid.New(),
 		EdgeRouter:  edgeRouter,
 		Router:      router,
 		send:        make(chan *channel2.Message, sendBufferSize),
 		closeNotify: make(chan struct{}, 0),
-		running:     concurrenz.AtomicBoolean(0),
-		stopping:    concurrenz.AtomicBoolean(0),
+		running:     concurrenz.AtomicBoolean(1),
```
Comment (andrewpmartinez) [inline controller/sync_strats/rtx.go:52]: Defaults to running now

**Review by plorenz — APPROVED** [review]

**Pull request merged**